selfdrive/ui/paint.cc

```diff
@@ -108,6 +108,23 @@ static void draw_chevron(UIState *s, float x_in, float y_in, float sz,
   nvgRestore(s->vg);
 }
 
+static void draw_lead(UIState *s, float d_rel, float v_rel, float y_rel){
+    const UIScene *scene = &s->scene;
```

Comment:
scene is unused in this function.

Reply:
didn't know if this is needed. The other functions had it. Thanks for the explaination